sp20/syllabus.yml

```diff
@@ -1,17 +1,19 @@
 meeting00:
   required:
-    title: ''
-    cover: ''
-    filename: ''
-    instructors: []
-    description: ''
+    title: 'Starting with the Basics, Regression'
+    cover: 'https://pbs.twimg.com/media/Dqb-Nk_XcAAm1_j.jpg'
+    filename: 'linear-regression'
+    instructors: ["JarvisEQ", "bb912"]
+    description: "You always start with the basics, and with AI it's no different!
+    We'll be getting our feet wet with some simple, but powerful, models and 
+    demonstrate their power by applying them to real world data."
   optional:
     date: 01-22
     room: HPA1 112
-    tags: []
+    tags: ["linear-regression", "basics", "logistic-regression", "regression"]
     papers: null
     urls:
-      slides: ''
+      slides: 'https://drive.google.com/open?id=1oDo8_RpLqBdq5BcE-mWFGkKAsxY5cns5FN19hZDl42s'
```

Comment:
this should be the google slides link - like... https://docs.google.com/d/presentation/... – it can either be that link or the really long ID after that (should be between .../presentation/<this-is-the-ID>/edit when you look at the URL).